core/trino-spi/src/main/java/io/trino/spi/function/InvocationConvention.java

```diff
@@ -63,7 +63,7 @@ public boolean supportsSession()
         return supportsSession;
     }
 
-    public boolean supportsInstanceFactor()
+    public boolean supportsInstanceFactory()
```

Comment:
public method name change in SPI should be part of release note.

Reply:
This is for engine consumption only, it shouldn't be viewed as part of SPI in practice.